en/finding-sorting-and-cleaning-entries/groups.md

```diff
@@ -177,6 +177,12 @@ The "groups" column is displayed by default. Using the menu **Options → Prefer
 
 ![Preferences for tab Entry table](https://user-images.githubusercontent.com/6931104/188165479-1beeed58-638b-4ef8-97d2-70bd350753fa.png)
 
+## Adding articles to groups
```

Comment:
The paragraph you introduce is a duplicate of "Creating a group and adding entries to it", which can be found on the same page in line 24-30. I would suggest to adjust and refine that paragraph instead of creating a new one.

When you do so, keep in mind, "article" is a very specific Bibtex/Biblatex entry type, but there are multiple entry types that can be added to groups, such as `@book`, `@periodical`, `@patent`, ... (See e.g. [Biblatex Cheatsheet](https://tug.ctan.org/info/biblatex-cheatsheet/biblatex-cheatsheet.pdf)). You can list some examples for entry types to make it more clear or make it more specific, that articles can be added this way, but ultimately, they cannot completely replace an overarching term like "entrytype".

Reply:
I guess I don't think it is a duplicate. The process of _creating_ a group and adding entries is different from adding entries after-the-fact to an already created group; and reading the content (as I did when I was looking for this information), it's not the same information. Would an adjusted proposed title for this section help?

Reply:
I adjusted the title of the section to clarify its difference from the paragraph you specified.

Reply:
Could you move your new paragraph directly to the lines succeeding lines 24-30? Even better would be to incorporate your information in this paragraph as a subparagraph, no? I think there is no need to make a new paragraph, but if you insist on making a new paragraph on how to add entries to groups, then I would suggest to rewrite paragraph "Creating a group and adding entries to it" and move all the information about how to add entries to already existing groups into this new paragraph (and this new paragraph should be in close proximity to it).

In short: We would have two new paragraphs:

1. The first for "Creating groups"
2. The second for "How to add entries to groups"

The problem is this: There is in fact a way to add entries to groups that creates the group at the same time. If you go to `{}bibtex source` general tab, you can edit the groups field and write text in there, which automatically creates a new group, but this new group will not show in the user interface (UI). Anyway, since this is possible, I still think it would be good to have all information about how to create and add entries to groups in one paragraph... You can make multiple subparagraphs.